README.md

```diff
@@ -148,9 +150,9 @@ If you need to debug the output of the dockers after the tests you can request t
 
 ## Docs
 
-[Tool Developer Guide](http://docs.codacy.com/v1.5/docs/tool-developer-guide)
```

Comment:
We want this to point to the seed and not the docs anymore. The source of truth should be GitHub.